common/utils/src/main/resources/error/error-conditions.json

```
@@ -1093,6 +1093,12 @@
     ],
     "sqlState" : "22008"
   },
+  "DATE_TIME_FIELD_OUT_OF_BOUNDS" : {
```

We use the term `DATETIME` in most cases. For instance, look at the error condition above:
```suggestion
  "DATETIME_FIELD_OUT_OF_BOUNDS" : {
```